rogue-thi-app/lib/backend/authenticated-api.js

```diff
@@ -93,6 +94,14 @@ export class AuthenticatedAPIClient extends AnonymousAPIClient {
     return resp
   }
 
+  getRoomDistances (room) {
+    if (!room) {
+      return {}
+    }
+
+    return roomDistances[room.toUpperCase()] || {}
+  }
```

Comment:
it does not make sense to have this as a method in the authenticated-api class. Just use the `roomDistances` data directly in rooms-utils.js